services/QuillLMS/spec/factories/activity_pack_release_units.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+FactoryBot.define do
```

Comment:
I don't think there's a model file for this one and it doesn't seem to show up anywhere else, is this supposed to be here?

Reply:
I suspect that this is leftover from an earlier draft of the code.  Note that it also reference `activity_pack_release` as an attribute, and I don't think that's a thing at the moment.  Good catch.